### ISSUE TITLE
Rendering breaks when slurping in a file under clerk/html

### ISSUE THREAD

**Issue by olymk2** [nextjournal/clerk#391]
I think this is specific to garden as it works locally, 

There is a broken version here.

https://github.clerk.garden/olymk2/clerk_inkscape_svg/commit/65f5aa8483b7aa29ffb5fb31eb34f65ab2fa838a/

You can see the repo I used below.

https://github.com/olymk2/clerk_inkscape_svg/blob/master/src/bb_inkscape_example.clj#L20

the slurp wrapped in a pre and clerk/html seem to be the cause, using slurp on it own is fine.
```
(clerk/html [:pre (slurp "src/bb_inkscape_example.sh")])
(slurp "src/bb_inkscape_example.sh")
```

The expected goal is to pull in an example code and have it displayed nicely in the notebook, in this example sh and inx files, if I don't use clerk/html pre then its all in one line and not very readable.

(clerk/read) could be an option for these situations, but regardless I don't think rendering should break, shout if you need more info.

**Comment by mk** [comment]
Thanks for filing the issue, Oliver. If you don't mind, it would be helpful to narrow it down a bit more to a minimal string that breaks the rendering.

**Comment by olymk2** [comment]
sure, I will try and reduce I actually tried doing it for an .sh file and a .inx file and both seemed to fail, but I will strip one of them down.

**Comment by olymk2** [comment]
Ignore the sh that seems to be working, the smallest example I can come up with is this.

`<script></script>`

Seems specific to this tag perhaps because it allows you to insert js in html so perhaps some special handling causing it ?

**Comment by mk** [comment]
Hey @olymk2, sorry it took me a while to get to this. I'm having trouble reproducing this:

<img width="1080" alt="Screen Shot 2023-01-31 at 20 48 42" src="https://user-images.githubusercontent.com/1187/215866881-a9860620-b6fb-48ef-87b2-5c31709cd6d7.png">

In the second example I would expect to see no result since an empty script tag doesn't render anything.

**Comment by mk** [comment]
This is an escaping issue. Encoding the doc to work around this e.g. using Base64 should be straightforward.

**Comment by mk** [comment]
I tried base64 encoding in b78960a12a7fd4e77389e2e7278064c202679dcd but didn't want to to deal with the size overhead.

In 62e0d81421ce8991c9d4e643c50b29718c6d6761 I went with an alternative that only replaces the closing script tag instead, see https://html.spec.whatwg.org/multipage/syntax.html#cdata-rcdata-restrictions.

@olymk2 Thanks again for taking the time to file this issue and get it down to a minimal repro!
